docs/docker-testnet.md

```diff
@@ -66,12 +66,12 @@ docker run -d -p 3013:3013 -e PEERS_ADDRESS_0=http://31.13.248.103:3013/ aetrnty
 
 ## Persisting Data
 
-To persist blockchain data and node keys between container runs, use [Docker volumes](https://docs.docker.com/engine/admin/volumes/volumes/). Replace `~/.aeternity/.chain` with location of your choice.
+To persist blockchain data and node keys between container runs, use [Docker volumes](https://docs.docker.com/engine/admin/volumes/volumes/). Replace `~/.aeternity/data` with location of your choice.
 
 
 ```bash
 docker run -d -p 3013:3013 \
-    -v ~/.aeternity/.chain:/home/epoch/node/.chain \
+    -v ~/.aeternity/data:/home/epoch/node/data \
```

Comment:
Last time I checked data dir contained some other files as well. We're using docker bind mounts and this information will be obfuscated see https://docs.docker.com/engine/admin/volumes/

Reply:
@dincho I read:
> Bind mounts ... rely on the host machine’s filesystem having a specific directory structure available.

Are you saying that this will not work unless the client created `~/.aeternity/data` upfront? I expect then that the `docker -v ...` errs if that folder is not present so that user will read the error message, create folder and re-run command? Hence it looks to me no change needed in doc.

Reply:
Nope, sorry for not being detailed here.

When bind mount is used as in `-v ~/.aeternity/data:/home/epoch/node/data` if there is any files/directories in `/home/epoch/node/data` in the container, they would become unaccessible.

Example/Test:
```bash
$ docker run -it --rm --entrypoint=bash aetrnty/epoch
epoch@8f8ac1e8959b:~/node$ ls data/
aecore  epoch_config_schema.json

$ mkdir /tmp/.aeternity && touch /tmp/.aeternity/blah
$ docker run -it --rm -v /tmp/.aeternity:/home/epoch/node/data --entrypoint=bash aetrnty/epoch
epoch@61f6fa7c7dae:~/node$ ls data/
blah
```

Reply:
> Last time I checked data dir contained some other files as well.

I see 
```
/tmp/node $ ls -d data/*
data/aecore                     data/epoch_config_schema.json   data/mnesia
```

Thanks for spotting this.

Reply:
> When bind mount is used as in -v ~/.aeternity/data:/home/epoch/node/data if there are any files/directories in /home/epoch/node/data in the container, they would become unaccessible.
> 
> Example/Test:

And thanks for clear example.